index.html

```diff
@@ -1,99 +1,157 @@
-            <p>
-              Contributor:
-              <a href="https://github.com/CodeLikeAGirl29">CodeLikeAGirl29</a>
-            </p>
-            <a href="Lioness/index.html">View Project</a>
-          </div>
+        
+        <div class="column links">
+        <h3>Some Links</h3>
+        
+         <ul>
+        
+          <li>
+           <a href="#faq">F.A.Q</a>
+          </li>
+          <li>
+           <a href="#cookies-policy">Cookies Policy</a>
+          </li>
+          <li>
+           <a href="#terms-of-services">Terms Of Service</a>
+          </li>
+          <li>
+           <a href="#support">Support</a>
+          </li>
+         </ul>
+        
         </div>
-        <div class="col-md-4">
-          <div class="project">
-            <h2>CodeBuddy</h2>
-            <p>
-              Contributor: <a href="https://github.com/beingsie">beingsie</a>
-            </p>
-            <a href="CodeBuddy/src/index.html">View Project</a>
-          </div>
+        
+        
+        <div class="column links">
+          <h3>Some Links</h3>
+           <ul>
+            <li>
+             <a href="#faq">F.A.Q</a>
+            </li>
+            <li>
+             <a href="#cookies-policy">Cookies Policy</a>
+            </li>
+            <li>
+            <a href="#terms-of-services">Terms Of Service</a>
+            </li>
+            <li>
+            <a href="#support">Support</a>
+            </li>
+          </ul>
         </div>
-        <div class="col-md-4">
-          <div class="project">
-            <h2>Portfolio</h2>
-            <p>
-              Contributor:
-              <a href="https://github.com/faizaslam11">faizaslam11</a>
-            </p>
-            <a href="Portfolio/index.html">View Project</a>
+        
+        <div class="column subscribe">
+         <h3>Newsletter</h3>
+          <div>
+           <input type="email" placeholder="Your email id here" />
+           <button>Subscribe</button>
           </div>
+        
         </div>
-        <div class="col-md-4">
-          <div class="project">
-            <h2>Books_Store</h2>
-            <p>
-              Contributor:
-              <a href="https://github.com/Bharath-KumarReddy">Bharath Kumar</a>
-            </p>
-            <a href="Books_store_LandingPage/index.html">View Project</a>
-          </div>
+        
         </div>
-        <div class="col-md-4">
-          <div class="project">
-            <h2>GymsoWebsite</h2>
-            <p>Contributor: <a href="https://github.com/Surieya">Surieya</a></p>
-            <a href="gymsoWebsite/index.html">View Project</a>
+        
+        <div class="row copyright">
+          <div class="footer-menu">
+        
+          <a href="">Home</a>
+          <a href="">About</a>
+          <a href="">Contact</a>
+          <a href="">Blog</a>
+          <a href="">Social</a>
+        
           </div>
+           <p>Copyright &copy; 2023 All rights reserved</p>
```

Comment:
this can be removed, keep the <p>Copyright &copy; 2023 All rights reserved</p> but add it to the bottom of the footer and center it an i think that will do it